### PR TITLE
[next_stage] Fix format of TestPlan

### LINKED ISSUE #6559: [next_stage] TestPlan not in md format
**Describe the bug**
The TestPlan file for next_stage module is not a .md file and is missing a header.

### PR DESCRIPTION
**Pull request by CamilleBeau**
## Brief summary of changes
Made file .md and added header so that it matches format of other testplans.
- [X] Have you updated related documentation?

#### Testing instructions (if applicable)

1.

#### Link(s) to related issue(s)

* Resolves #6559

### CODE REVIEW

File: modules/next_stage/test/TestPlan.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
    and for candidates at the user's site.
    [Automation Testing]
 4. Ensure that you get an error if dates do not match.
    [Automation Testing]
-5. Ensure that you get an error if scan_done missing
+5. Ensure that you get an error if `scan_done` missing
    [Manual Testing]
 6. Create a timepoint where the age (date entered minus DoB) falls within the
-   AgeMin and AgeMax for a test_battery. Ensure that those instruments are
+   AgeMin and AgeMax for a `test_battery`. Ensure that those instruments are
```
Comment (laemtl) [inline modules/next_stage/test/TestPlan.md:15]: AgeMin and AgeMax for a `test_battery`. Those test_battery needs to have a matching subprojectID and a NULL Visit_label. Ensure that those instruments are

File: modules/next_stage/test/TestPlan.md [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
    inserted (for the correct subprojectID).
```
Comment (laemtl) [inline modules/next_stage/test/TestPlan.md:16]: inserted.

File: modules/next_stage/test/TestPlan.md [hunk 3]
```diff
@@ -1,33 +1,34 @@
...
    [Manual Testing]
-7. Create a timepoint where there is a test_battery entry for the
-   Visit_label and start the stage. Ensure that those were inserted into
+7. Create a timepoint where there is a `test_battery` entry for the
+   `Visit_label` and start the stage. Ensure that those were inserted into
```
Comment (laemtl) [inline modules/next_stage/test/TestPlan.md:19]: `Visit_label` and subprojectID. Ensure that those were inserted into

File: modules/next_stage/test/TestPlan.md [hunk 4]
```diff
@@ -1,33 +1,34 @@
...
    the battery, and NOT the ones based on age.
```
Comment (laemtl) [inline modules/next_stage/test/TestPlan.md:20]: the battery, and NOT the ones based on age (when Visit_label is NULL).

File: modules/next_stage/test/TestPlan.md [hunk 5]
```diff
@@ -1,33 +1,34 @@
...
+8. Ensure that any instruments in `test_battery` with firstVisit='Y' where
    inserted if and only if it's the first timepoint started. Ensure that
    firstVisit='N' instruments are inserted if and only if it is NOT the
    first timepoint started, and firstVisit=null whether or not it's the
    first time point.
    [Manual Testing]
 9. Ensure that if a CenterID is set for a row in the test battery, the
-   instrument is inserted for candidates at that centerID (if Visit_label
-   or Age are correct.), and not other centers.
+   instrument is inserted for candidates at that centerID (if `Visit_label`
```
Comment (laemtl) [inline modules/next_stage/test/TestPlan.md:29]: instrument is inserted for candidates at that centerID (if instrument `Visit_label` is NULL or identical to the timepoint's one

File: modules/next_stage/test/TestPlan.md [hunk 6]
```diff
@@ -1,33 +1,34 @@
...
+   or `Age` are correct.), and not other centers.
```
Comment (laemtl) [inline modules/next_stage/test/TestPlan.md:30]: and `Age` is correct), and not other centers.

**Comment by CamilleBeau** [comment]
@laemtl Thank you! I updated it to reflect your suggestions.

**Pull request merged**